client/blocks/comment-detail/comment-detail-post.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ReaderAvatar from 'blocks/reader-avatar';
+
+export const CommentDetailPost = ( {
+	author,
+	siteIcon,
+	title,
+	url,
+} ) =>
+	<div className="comment-detail__post">
+		<ReaderAvatar
```

Comment:
Wait for that tho: this was actually my own addition, the current design iteration doesn't use any avatar there. 🙂 
I've just stumbled upon that component while browsing the design docs and thought it would look good here.